sidm/tools/sidm_processor.py

```diff
@@ -307,7 +307,7 @@ def build_cuts(self):
     def build_histograms(self):
         """Create dictionary of Histogram objects"""
         hist_menu = utilities.load_yaml(self.histograms_cfg)
-
+        
```

Comment:
Hi @mariajose2398, can you fix this extra whitespace real quick before I merge?

Reply:
Just did it